src/hot-keys/RaceSpecificHotkeys/hotkeyData/nightelf/buildings.ts

```diff
@@ -9,7 +9,9 @@ import {
     defaultFighterAbilities,
     defaultMainBuilding, rally, shop, stop
 } from "@/hot-keys/RaceSpecificHotkeys/hotkeyData/commonHotkeys";
+
 const uproot = Ability.Create('Uproot', 'btnuproot', 'aro1', 'R', [])
+const root = Ability.Create('Root', 'btnroot', 'aroo', 'R', [])
```

Comment:
There is a `Unhotkey` Ability, would fit better, so we have only one key.

Reply:
ye, you're right. I must have gotten a bit confused last night. I'll fix it!